client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -67,14 +67,13 @@
           </zrForm.Field>
         </Ui::Question>
       {{/if}}
-
-      {{#if this.zrAppendixF}}
+    
+    {{else if (eq this.zrTypeLabel 'ZR')}}
```

Comment:
I guess this is okay since we rely so heavily on the action type labels anyway in `lookup-action-type` helper.  But in general I think it would be less brittle if throughout the app we used codes as much as possible for business logic and comaprisons, instead of text labels.

Reply:
you mean use the lookup-action-type helper in the template here?

Reply:
Mm, I mean just compare against `this.args.zrForm.data.dcpZoningresolutiontype`. Or maybe introduce a getter to alias that property, called `zrTypeCode`, and compare against that instead. 
`(eq this.zrTypeCode (optionset 'affectedZoningResolution' 'actions' 'code' 'ZR'))`

Reply:
So we're using the optionset to determine `zrTypeLabel` in the javascript for this component. 

What I'm gathering is that you're saying that we should avoid using optionsets in computed properties and then employing them in templates? And instead we should always just try to use the optionset directly in the template instead? 

I can try to update other places if you think that's best, but could you also explain why using directly in the template is better

Reply:
Think of there being two different languages: machine codes (717000) and human-speak labels ("ZR").
My recommendation is work with machine codes as much as possible throughout the code, especially for any business logic like comparisons (regardless of in template or a computed). We would only use the human-speak labels when printing them for **display** in the templates.

Reply:
Okay so what I'm taking from this is that using optionsets in computed properties and then employing those in templates is a no-go in the future. And that if we can, we should always be using optionsets DIRECTLY in the template when possible